### DISCUSSION TITLE
Multiple Root CAs with one instance of EJBCA?

### DISCUSSION THREAD

**Discussion by kpeirce** [Keyfactor/ejbca-ce#463] [Q&A] [answered]
Does EJBCA support multiple Root CAs with a single instance of EJBCA? If it does, am I better off using multiple instances? I could not find a discussion  on this topic ("multiple CAs").

**Comment by primetomas** [comment, marked as answer]
Yes you can have as many Roots and Subs as you like in a single instance. Weather you want to use separate instances or not is not a technical question. That would be down to the respective Root CA policies, if one require specific physical isolation that others don't, if there are different people managing the different Root CAs, etc.
Technically, no problem.

**Reply by kpeirce** [reply]
Thanks for the amazingly quick reply!
